app/src/main/res/values/strings.xml

```diff
@@ -149,4 +149,8 @@ We engage our community by contributing to open source, collaborating with the g
     <string name="url_terms">https://anitab.org/terms-of-use/</string>
     <string name="url_privacy">https://anitab.org/privacy-policy/</string>
     <string name="url_code_of_conduct">https://ghc.anitab.org/code-of-conduct/</string>
+    <string name="task_update_confirm">Update task</string>
+    <string name="task_update_title">Complete task?</string>
+    <string name="task_update_message">Are you sure you want to task:\u0020</string>
```

Comment:
Can you please make this as a formatted string that receives the description as a parameter?
Does this make sense to you? instead of creating a separate string for the `?` sign?

Reply:
Also could it be like "Are you sure you want to complete this task"?